video/out/opengl/context_angle.c

```diff
@@ -109,7 +109,7 @@ struct priv {
 
     int sc_width, sc_height; // Swap chain width and height
     int swapinterval;
-    bool flipped;
+    bool OPT_BOOLped;
```

Comment:
I think find/replace got wonky here.

Reply:
And of course the one instance where this happens is in a file that doesn't get compiled on linux :laughing:
Thanks